### ISSUE TITLE
Chesapeake7 and Chesapeake13 can't have the same root directory

### ISSUE THREAD

**Issue by calebrob6** [microsoft/torchgeo#419]
```
ds13 = Chesapeake13(root="data/", download=True)
ds7 = Chesapeake7(root="data/", download=True)
```

will not throw any errors, but when I sample from ds7, then RasterDataset will use the `Baywide_13Class_20132014.tif` file

**Comment by adamjstewart** [comment]
This should be really easy to fix, we just need to add something like:
```python
filename_glob = "Baywide_13Class_20132014.tif"
```
to each dataset.

**Comment by calebrob6** [comment]
Yep, fixed in #417
